src/tools/clippy/tests/ui/range_plus_minus_one.fixed

```diff
@@ -28,30 +28,30 @@ fn main() {
     for _ in 0..2 {}
     for _ in 0..=2 {}
 
-    for _ in 0..=3 {}
+    for _ in 0..=3  {}
     for _ in 0..=3 + 1 {}
 
-    for _ in 0..=5 {}
+    for _ in 0..= 5 {}
```

Comment:
Those additional whitespaces in the fixed code, look weird.

Reply:
Suggestions already have the assumption that `rustfmt` will be run afterwards. That spacing is because of how the spans work out. It's not great, but it is preferable to have slightly less pretty output for more robust spans.

Reply:
I agree, we also assume in Clippy, that everyone runs `rustfmt`, so this is fine. Just wanted to point it out, if it was an oversight.